Source/Meadow.Foundation.Peripherals/Sensors.Distance.Me007ys/Samples/Me007ys_Sample/MeadowApp.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Meadow;
+using Meadow.Devices;
+using Meadow.Foundation.Sensors.Distance;
+using Meadow.Units;
+using System;
+using System.Threading.Tasks;
+
+namespace Me007ys_Sample
+{
+    // Change F7FeatherV2 to F7FeatherV1 for V1.x boards
+    public class MeadowApp : App<F7CoreComputeV2>
+    {
+        Me007ys me007ys;
+
+        public override Task Initialize()
+        {
+            Resolver.Log.Info("Initialize...");
+
+            me007ys = new Me007ys(Device, Device.PlatformOS.GetSerialPortName("COM4"));
+
+            var consumer = Me007ys.CreateObserver(
+                handler: result =>
+                {
+                    Resolver.Log.Info($"Observer: Distance changed by threshold; new distance: {result.New.Centimeters:N2}cm, old: {result.Old?.Centimeters:N2}cm");
+                },
+                filter: result =>
+                {
+                    if (result.Old is { } old)
+                    {
+                        return Math.Abs((result.New - old).Centimeters) > 0.5;
+                    }
+                    return false;
+                }
+            );
+            me007ys.Subscribe(consumer);
+
+            me007ys.DistanceUpdated += Me007y_DistanceUpdated;
+
+            return Task.CompletedTask;
+        }
+
+        public override async Task Run()
+        {
+            var distance = await me007ys.Read();
```

Comment:
Shouldnt this be calling `MeasureDistance`? I dont see a Read method, must be on ByteCommSensorBase

Reply:
yeah its on the base class

Reply:
Then why do we need a `MeasureDistance` method? Seems redundant.